src/preferences/node/json-preference-server.ts

```diff
@@ -39,9 +39,10 @@ export class JsonPreferenceServer implements IPreferenceServer {
     /**
      * Checks to see if the preference file was modified
      */
-    protected arePreferencesAffected(event: FileChangesEvent): boolean {
-        return event.changes.some(c => {
-            return (c.uri === this.preferencePath.toString() && c.type === FileChangeType.UPDATED);
+    protected arePreferencesAffected(changes: FileChange[]): boolean {
+
+        return changes.some(c => {
+            return (c.uri === this.preferencePath && c.type === FileChangeType.UPDATED);
```

Comment:
`c.uri.toString() === this.preferencePath.toString()`, otherwise you compare 2 references